contracts/mock/Foreign20.sol

```diff
@@ -61,6 +61,10 @@ contract Foreign20 is ERC20Pausable, MockNativeMetaTransaction {
     function destruct() public {
         selfdestruct(payable(msg.sender));
     }
+
+    function allowance(address owner, address spender) public view override returns (uint256) {
```

Comment:
This function seems a bit unnecessary since it does not change `allowance` behaviour.

But since it's in a mock, I don't really mind.